src/main/java/unc/Duke.java

```diff
@@ -0,0 +1,198 @@
+package unc;
+
+import unc.task.Deadline;
+import unc.task.Event;
+import unc.task.Task;
+import unc.task.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.*;
```

Comment:
Could consider breaking up this import statement into multiple explicit statements.